docs/index.html

```diff
@@ -469,18 +469,25 @@ <h5>Platform-specific</h5>
 
         <dt id="faq-platform-mobile"><a href="#faq-platform-mobile">Does KeePassXC work on mobile phones? If not, which app would you recommend?</a></dt>
         <dd>
-            No, KeePassXC only works on desktop systems. Porting it properly to mobile platforms would require a full rewrite.
-            You may be able to compile it for the mobile OS of your choice, but KeePassXC isn't at all optimized
+						We don't have our own mobile app, but you can have the same functionality on both Android and iOS!
+						<br>
+						For Android, we recommend <a href="https://play.google.com/store/apps/details?id=keepass2android.keepass2android">KeePass2Android</a>.
+						<br>
+						(it's <a href="https://github.com/PhilippC/keepass2android">open-sourced on GitHub</a>,
+						<a href="https://github.com/keepassxreboot/keepassxc/issues/1060#issuecomment-442347384">works for kdbx4 databases</a>
+						and even supports YubiKeys via NFC!)
+						<br>
+						And for iOS, we suggest <a href="https://itunes.apple.com/us/app/strongbox-password-safe/id897283731">Strongbox</a>
+						(also <a href="https://github.com/mmcguill/StrongBox">open-sourced on GitHub</a>!) 
+						<br>
+            For KeePassXC, porting it properly to mobile platforms would require a full rewrite.
+            You may be able to compile KeePassXC for the mobile OS of your choice, but it isn't at all optimized
             for mobile screen sizes and form factors, let alone multi-touch input. We also don't see any advantage in
-            providing a mobile version of KeePassXC when there are excellent alternatives, already. For Android we
-            recommend <a href="https://play.google.com/store/apps/details?id=keepass2android.keepass2android">KeePass2Android</a>
-            and for iOS <a href="https://itunes.apple.com/us/app/strongbox-password-safe/id897283731">Strongbox</a>. Other
-            alternatives are <a href="https://itunes.apple.com/us/app/minikeepass/id451661808?mt=8">MiniKeePass</a>
+						providing a mobile version of KeePassXC when there are excellent alternatives (as mentioned above), already. 
+						<br>
+            Other alternatives are <a href="https://itunes.apple.com/us/app/minikeepass/id451661808?mt=8">MiniKeePass</a>
             (appears abandoned, no KDBX4 support as of now) and <a href="https://itunes.apple.com/us/app/keepass-touch/id966759076?mt=8">KeePass
```

Comment:
I wouldn't even mention minikeepass, it is abandoned.

Reply:
Should we mention `KeePass Touch` or just remove the `Other alternatives` section?

Reply:
Just remove it. Strongbox seems to be the only viable alternative at this time.